### DISCUSSION TITLE
How to support ExpandedNodeId?

### DISCUSSION THREAD

**Discussion by Neyzoter** [gopcua/opcua#524] [Q&A] [answered]
I use `examples/browse/browse.go` to browse a OPCUA Server WITH ExpandedNodeId, but browse.go return timeout error.

I found that browse.go use [NodeId](https://github.com/gopcua/opcua/blob/main/ua/node_id.go) rather [ExpandedNodeId](https://github.com/gopcua/opcua/blob/main/ua/expanded_node_id.go).

**Comment by magiconair** [comment, marked as answer]
Feel free to copy and adjust the example as you see fit.
